### DISCUSSION TITLE
floating label background wrong when input is disabled

### DISCUSSION THREAD

**Discussion by FVZGmbH** [twbs/bootstrap#38867] [General] [answered]
Hi,
today I switched from 5.22 to 5.3 and my labels background went wrong.
I use floating labels, but when I set disabled attr @ input field, the field itself went grey but the label stay white.
I can overwrite that, but I think its not the way it should be.

**Comment by patkon** [comment, marked as answer]
Hi @FVZGmbH,
it's a confirmed issue and will surely be fixed with the next release.
https://github.com/twbs/bootstrap/issues/38731
